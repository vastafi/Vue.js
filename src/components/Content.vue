<template>
<div>
  Content
</div>
</template>

<script>
export default {
  name: "Content"
}
</script>

<style scoped>

</style>