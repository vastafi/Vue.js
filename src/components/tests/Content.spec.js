import Content from "../Content";
import {createLocalVue, shallowMount} from "@vue/test-utils";
import VueRouter from 'vue-router'

const localVue = createLocalVue();
localVue.use(VueRouter)

localVue.component('RouterView', {
    functional: true,
    render:() => 'test',
})

localVue.component('v-snackbar',{
    functional: true,
    render: ()=>'test'
})

describe('Content',()=> {
    it('should mount',()=> {
        const component = shallowMount(Content,{
           localVue
        });
        expect(component.element).toMatchSnapshot();
    })
})