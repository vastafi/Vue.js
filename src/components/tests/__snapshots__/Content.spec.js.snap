// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Content should mount 1`] = `
<v-main>
  <v-container>
    <routerview-stub />
     
    <errormessage-stub />
  </v-container>
</v-main>
`;
